druid/src/lib.rs

```diff
@@ -57,7 +57,7 @@ pub use command::{sys as commands, Command, Selector};
 pub use data::Data;
 pub use env::{Env, Key, Value};
 pub use event::{Event, WheelEvent};
-pub use lens::{Lens, LensWrap};
+pub use lens::{Field, Lens, LensWrap};
```

Comment:
do we need to export the `lens!` macro itself? I'm not sure how macro visibility works now. 😕

Reply:
also do we expect `Field` to ever be used directly, or is it an implementation detail of `lens!`? If the latter I'd probably not reexport here.

Reply:
> do we need to export the lens! macro itself? I'm not sure how macro visibility works now.

Declarative macros annotated with `#[macro_export]` are always implicitly exported from the crate root.

> also do we expect Field to ever be used directly, or is it an implementation detail of lens!?

It might be a useful building block for things that are difficult or impossible to express with `lens!` for some reason, e.g. something that implements copy-on-write semantics in the mutable getter. In any case, macro_rules can't reference things that aren't public, so the best we could only hide it in the docs.

Reply:
Okay, thanks for the explanations.